src/ethereum_test_specs/blockchain.py

```diff
@@ -453,6 +453,123 @@ def discard_fixture_format_by_marks(
             return True
         return False
 
+    @classmethod
+    def from_fuzzer(
+        cls,
+        fuzzer_output: Dict[str, Any],
```

Comment:
```suggestion
        fuzzer_output: FuzzerOutput,
```
And parse when calling.